.github/workflows/jq.yml

```diff
@@ -3,10 +3,14 @@
 on:
   push:
     paths:
-      - '**.json'
+      - ${{ env.paths }}
   pull_request:
     paths:
-      - '**.json'
+      - ${{ env.paths }}
+  workflow_dispatch:
+
+env:
+  paths: assets/json/*.json
```

Comment:
`env` is not for defining a variable but for defining environment variable so it should not be used like this.

Reply:
However, github action don't support yaml's anchor, I don't know how to realize this function except env?

Reply:
Yes. So we don't do that.